src/client/app/containers/CompareChartContainer.js

```diff
@@ -26,6 +26,22 @@ function mapStateToProps(state) {
 	// Power used up to this point last week
 	let currentLastWeek = 0;
 	const soFar = moment().diff(moment().startOf('week'), 'days');
+	const delta = function (change) {
+		if (isNaN(change)) {
+			return '';
+		}
+		if (change < 0) {
+			return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% less energy this week.`;
+		}
+		return `You have used ${parseInt(change.toFixed(2).replace('.', ''), 10)}% more energy this week.`;
+	};
+
+	const colorize = function (change) {
+		if (change < 0) {
```

Comment:
Shouldn't this be <= 0? Or is no change not great?

Reply:
We want them to improve!